pkg/test/istio.io/README.md

```diff
@@ -1,13 +1,42 @@
-# Istio.io Example Tests
+# Istio.io preliminary test cases
+
+Welcome to the development of the istio.io preliminary test cases!
+
+This work is experimental. This work is also under construction and may not work
+for you. The Istio developers have done some work to document known problems or
+workarounds. We expect this work will take several releases to complete.
 
 ## Overview
-This folder contains tests for the different examples on Istio.io.
+This folder contains tests for the different examples on [istio.io](https://istio.io).
+The goal of this work is to ensure [istio.io](https://istio.io) is tested according
+to the code in the [Istio repository](https://github.com/istio).
 
 ## Writing Tests
 
+This section needs fleshing out.
 
 ## Executing Tests Locally using Minikube
-* Create a Minikube environment as advised on istio.io
-* Run the tests as follows:
- go test ./... -p 1 --istio.test.env kube --istio.test.kube.config ~/.kube/config -v
 
+- Create a [Minikube environment](https://preliminary.istio.io/docs/setup/kubernetes/platform-setup/minikube/)
```

Comment:
nit/personal preference - we should encourage https://kind.sigs.k8s.io for testing purposes rather than minikube

Reply:
we don't have docs for setup.  I agree though, kind is preferrable.

Reply:
To create a cluster named `test` with 2 workers:
```
kind create cluster --name=test --config=- <<EOF
kind: Cluster
apiVersion: kind.sigs.k8s.io/v1alpha3
nodes:
- role: control-plane
- role: worker
- role: worker
EOF
```
Then replace `~/.kube/config` with `$(kind get kubeconfig-path --name="test")`.

Reply:
To delete the cluster:
```
kind delete cluster --name=test
```

Reply:
@rlenglet Thanks for the `kind` instructions. I just tried them in place of my Docker for Mac and things worked except for some upcoming changes from @brian-avery (issue with scripts not using the specified kubeconfig).

Reply:
That's weird. This worked with my test. Maybe that's because I also set `$KUBECONFIG`?